Domains/4-Application/OpenSite/OpenSite.ecschema.xml

```diff
@@ -1108,4 +1112,18 @@
 
         <ECProperty propertyName="ControlVertexSlope" typeName="double" category="SurfaceControl_Vertex" kindOfQuantity="rru:SLOPE" displayLabel="Slope" description="Slope defined by a control vertex override." />
     </ECEntityClass>
-</ECSchema>
+
+    <ECEntityClass typeName="SurfaceControlLowPoint" modifier="Sealed" displayLabel="Control Low Point" description="Defines a location point which controls low point control object.">
+        <BaseClass>SurfaceControlVertex</BaseClass>
+
+        <ECProperty propertyName="ControlSlope" typeName="double" category="SurfaceControl_Vertex" kindOfQuantity="rru:SLOPE" displayLabel="Slope" description="Slope defined by a control low point override." />
+        <ECProperty propertyName="MaximumControlRadius" typeName="double" category="SurfaceControl_Vertex" kindOfQuantity="rru:LENGTH" displayLabel="Radius" description="Max radius defined by a control low point override." />
+        <ECProperty propertyName="RelatedElementId" typeName="string" category="SurfaceControl_Vertex" displayLabel="Related Element" description="Element id related to a control low point" />
```

Comment:
We should not store Element Ids in such "invisible" way. Perhaps you want to use a <ECNavigationProperty here?